e2e/weakly_connected_components_test/test_online_adding_components/test.yml

```diff
@@ -0,0 +1,54 @@
+- query: >-
+    CALL weakly_connected_components.get() YIELD node, component_id
+    RETURN node.id AS node_id, component_id
+    ORDER BY node_id, component_id
+  output:
+    - node_id: 0
+      component_id: 0
+    - node_id: 1
+      component_id: 0
+    - node_id: 2
+      component_id: 0
+    - node_id: 3
+      component_id: 1
+    - node_id: 4
+      component_id: 1
+    - node_id: 5
+      component_id: 1
+- query: >-
+    CALL weakly_connected_components.get() YIELD node, component_id
+    RETURN node.id AS node_id, component_id
+    ORDER BY node_id, component_id
+  output:
+    - node_id: 0
+      component_id: 0
+    - node_id: 1
+      component_id: 0
+    - node_id: 2
+      component_id: 0
+    - node_id: 3
+      component_id: 1
+    - node_id: 4
+      component_id: 1
+    - node_id: 5
+      component_id: 1    
+    - node_id: 6
+      component_id: 2
+- query: >-
+    CALL weakly_connected_components.get() YIELD node, component_id
+    RETURN node.id AS node_id, component_id
+    ORDER BY node_id, component_id
+  output:
+    - node_id: 0
+      component_id: 0
+    - node_id: 1
+      component_id: 0
+    - node_id: 2
+      component_id: 0
+    - node_id: 3
+      component_id: 1
+    - node_id: 4
+      component_id: 1
+    - node_id: 5
+      component_id: 1    
+      
```

Comment:
Empty line one too much?

Reply:
Yup, usually I let these to formatter, but do not have one for YAML